.github/workflows/build_publish_wheels.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Publish the Kratos Wheels
+
+# on: # build the corresponding release branch
+#     release:
+#       types: [created]
+
+on: push
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Pull docker image
+      run: sudo docker pull kratosmultiphysics/kratos-wheelbuilder-linux
+
+    - name: Build wheels
+      run: |
+        mkdir wheels
+        sudo docker run --rm -v ${GITHUB_WORKSPACE}/wheels:/workspace/out kratosmultiphysics/kratos-wheelbuilder-linux master 2
+
+  windows:
+    runs-on: windows-latest
+
+    steps:
+    - name: Pull docker image
+      run: docker pull kratosmultiphysics/kratos-wheelbuilder-windows
+
+    - name: Build wheels
+      shell: pwsh
+      run: |
+        mkdir wheels
+        docker run --rm --memory=4 --cpus=2 -v ${GITHUB_WORKSPACE}\wheels:c:\out kratosmultiphysics/kratos-wheelbuilder-windows master
```

Comment:
@HubertBalcerzak I think I am again doing sth wrong here :sweat_smile: 
I get the following error:
`C:\Program Files\Docker\docker.exe: Error response from daemon: invalid volume specification: '\wheels:c:\out'.`
could you pleas have a look

(or maybe @roigcarlo or @oberbichler know what I di wrong?)

Reply:
@philbucher Well, you have to run windows build on a windows runner.
`runs-on: windows-latest`
And the job script itself must be in powershell. I've been experimenting some time ago, this is the result:
https://github.com/HubertBalcerzak/Kratos/blob/master/.github/workflows/generate-wheels-windows.yml
```
    - name: Run a one-line script
      run: |
        New-Item -ItemType directory -Path wheels
        docker run --rm --memory=6G --cpus=2 -v wheels:c:\out hubertus248/kratoswheelbuilder-win master
        Compress-Archive -Path wheels -DestinationPath wheels-windows.zip
    - uses: actions/upload-artifact@v1
      with:
        name: wheels-windows
        path: wheels-windows.zip
```

Reply:
oh, you already are running on windows, my bad. Then it's a problem with that path variable. It would be fine to use just use local path wheels:c\out

Reply:
ok thx let me try

Reply:
now it works